pkg/utils/wrappers.go

```diff
@@ -0,0 +1,126 @@
+package utils
```

Comment:
Yeah, was thinking about it but I'd leave it for the future when we'll be wanting to reuse this code from `keep-core`. It may happen that we do something in the meantime that will invalidate this idea.

Reply:
Sure. Let's leave this for now then.